### PR TITLE
Added Episode of care valuesets

### PR DESCRIPTION
**Pull request by Gokulsignal**
@enjoysparkling , Hi Matt, Kindly review the pull request.

### CODE REVIEW

File: input/vocabulary/valueset-episode-of-care-type.json
Comment (enjoysparkling) [inline input/vocabulary/valueset-episode-of-care-type.json:1]: Changes required:
```
Message  : Instance failed constraint vsd-10 "Must have a system if a code is present"
FilePath : input\vocabulary\valueset-episode-of-care-type.json
Id       : vsd-10
Severity : Error
Context  : ValueSet.expansion[0].contains[0]
```
- This should probably represent the Service Type from the HealthcareService resource.  Keep this and include that code system. Recorded in #71
- We may also want to include Program...

File: input/vocabulary/valueset-diagnosis-role.json
Comment (enjoysparkling) [inline input/vocabulary/valueset-diagnosis-role.json:1]: Changes required:
```
Message  : Instance failed constraint vsd-10 "Must have a system if a code is present"
FilePath : input\vocabulary\valueset-diagnosis-role.json
Id       : vsd-10
Severity : Error
Context  : ValueSet.expansion[0].contains[0]
```

**Review by enjoysparkling — CHANGES_REQUESTED** [review]
Need System for value set expansion codes

File: input/vocabulary/valueset-episode-of-care-type.json
Comment (arunsundar-m) [reply input/vocabulary/valueset-episode-of-care-type.json:1]: @enjoysparkling Should we ignore this valueset and bind healthcare-service-type valueset here? also we don't get your comment on program?

File: input/vocabulary/valueset-diagnosis-role.json
Comment (Gokulsignal) [reply input/vocabulary/valueset-diagnosis-role.json:1]: Updated the System to the expansion.contains on valueset-diagnosis-role

**Review by enjoysparkling — APPROVED** [review]
Looks good.

**Pull request merged**